pkg/client/client_test.go

```diff
@@ -2199,1464 +2205,1759 @@ U5wwSivyi7vmegHKmblOzNVKA5qPO8zWzqBC
 		})
 	})
 
```

Comment:
I had to move the List tests out, but with a bit of creative diff'ing it should be okay'ish to review